### ISSUE TITLE
no "hifi FBX" export in blender 2.81a

### ISSUE THREAD

**Issue by SilverfishVR** [Menithal/Blender-Hifi-Addon#22]
after upgrading to current stable (2.81a)  hifi FBX  export no longer works for me.
```
Traceback (most recent call last):
  File "C:\Users\Silverfish\AppData\Roaming\Blender Foundation\Blender\2.81\scripts\addons\hifi_tools\ext\modified_fbx_tools\__init__.py", line 360, in execute
    return mod_export_fbx_bin.save(self, context, **keywords)
  File "C:\Users\Silverfish\AppData\Roaming\Blender Foundation\Blender\2.81\scripts\addons\hifi_tools\ext\modified_fbx_tools\mod_export_fbx_bin.py", line 1041, in save
    ret = save_single(operator, context.scene, depsgraph, filepath, **kwargs_mod)
  File "C:\Users\Silverfish\AppData\Roaming\Blender Foundation\Blender\2.81\scripts\addons\hifi_tools\ext\modified_fbx_tools\mod_export_fbx_bin.py", line 953, in save_single
    False, media_settings, use_custom_props,
TypeError: __new__() missing 1 required positional argument: 'use_custom_props'

location: <unknown location>:-1
```

**Comment by Menithal** [comment]
Thanks, Ill look into it after the xmas holiday.

**Comment by Menithal** [comment]
Fixed in the 2.0 version.
